catalog.json

```diff
@@ -1,18 +1,18 @@
 {
   "id": "8478b533-2b59-4007-8494-2feec5970f94",
   "name": "cdn-route",
-  "description": "Custom URL, CDN, and free TLS certificates",
+  "description": "Custom domain, CDN caching, and TLS certificates",
   "bindable": true,
   "metadata": {
-    "displayName": "Content Distribution Network",
-    "documentationUrl": "https://cloud.gov/docs/services/cdn-route"
+    "displayName": "CDN Route",
```

Comment:
Hmm. Not sure what it should say. The things the users probably care about, in order:

1. Being the low-friction enabler for custom domains
1. Automatic certificates
1. Caching (the CDN part)